lib/parse.js

```diff
@@ -44,7 +44,9 @@ module.exports = function parseSentence(emitter, sentence, options) {
 
   if (valid === false) {
     emitter.emit('warning', { message: `Sentence "${sentence.trim()}" is invalid` })
-    return Promise.reject(new Error(`Sentence "${sentence.trim()}" is invalid`))
+    // FIXME: Do we really want an exception (or Promise.reject) here?
```

Comment:
Comments welcomed here: Should we return null or throw an exception when the checksum is invalid?

Reply:
null